napari/layers/points/points.py

```diff
@@ -1441,34 +1393,23 @@ def _slice_data(
         -------
         slice_indices : list
             Indices of points in the currently viewed slice.
-        scale : float, (N, ) array
-            If in `out_of_slice_display` mode then the scale factor of points, where
-            values of 1 corresponds to points located in the slice, and values
-            less than 1 correspond to points located in neighboring slices.
         """
         # Get a list of the data for the points in this slice
         not_disp = list(self._dims_not_displayed)
         indices = np.array(dims_indices)
         if len(self.data) > 0:
-            if self.out_of_slice_display is True and self.ndim > 2:
-                distances = abs(self.data[:, not_disp] - indices[not_disp])
-                sizes = self.size[:, not_disp] / 2
-                matches = np.all(distances <= sizes, axis=1)
-                size_match = sizes[matches]
-                size_match[size_match == 0] = 1
-                scale_per_dim = (size_match - distances[matches]) / size_match
-                scale_per_dim[size_match == 0] = 1
-                scale = np.prod(scale_per_dim, axis=1)
-                slice_indices = np.where(matches)[0].astype(int)
-                return slice_indices, scale
-            else:
-                data = self.data[:, not_disp]
-                distances = np.abs(data - indices[not_disp])
-                matches = np.all(distances <= 0.5, axis=1)
-                slice_indices = np.where(matches)[0].astype(int)
-                return slice_indices, 1
+            # TODO: is this np.maximum ok? zero thickness slice is probably unwanted
+            # (might as well set the layer invisible) and it also can cause issues with empty arrays
+            # so here we use a reasonable default (same as used to in napari before #3997)
+            not_disp_thickness = np.maximum(
+                np.array(thickness)[not_disp], 1e-5
+            )
+            distances = abs(self.data[:, not_disp] - indices[not_disp])
```

Comment:
```suggestion
            distances = np.abs(self.data[:, not_disp] - indices[not_disp])
```
not necessary, but maybe helpful for clarity? (just `abs` does work because the `ndarray` class defines `__abs__`)

Reply:
yup, that was just a typo, I definitely prefer the np version.